examples/README

```diff
@@ -21,17 +21,17 @@ Below is a brief description of the different directories found here:
   * axisartist - Examples related to the axisartist toolkit.
 
   * event_handling - How to interact with your figure, mouse presses,
-    key presses, object picking, etc.
+    keypresses, object picking, etc.
 
   * misc - Miscellaneous examples.  Some demos for loading and working
     with record arrays.
 
   * mplot3d - 3D examples.
 
-  * pylab_examples - The interface to Matplotlib similar to MATLAB.
+  * pylab_examples - The interface to Matplotlib is similar to MATLAB.
```

Comment:
I think this change is incorrect, only the pylab/pyplot interface to Matplotlib shares the global state of the "current" Figure and Axes.   While I agree this edit is more grammatically correct (the right hand side is now a sentence!), I think that it is OK because the right hand side does not need to be a sentence.

Reply:
This subdirectory does not exist any more either.